redaxo/src/core/tests/sql/sql_table_test.php

```diff
@@ -53,7 +53,7 @@ public function testCreate()
 
         $this->assertTrue($this->createTable()->exists());
 
-        rex_sql_table::clearInstance(self::TABLE);
+        rex_sql_table::clearInstance([1, self::TABLE]);
```

Comment:
das ist ein bc break, oder?

Reply:
Theoretisch ja, wobei ich mich frage, ob diese Methoden aus dem Instance-Cache-Trait wirklich zur Public-Api gehören sollten, oder nicht eher ein Implementierungsdetail sind, und nur deswegen public, weil teils anderen Klassen (im selben Addon etc.) von außen den Cache löschen können müssen.
Der User sollte mit diesen Methoden eigentlich nichts zu tun haben.

Ich hatte noch die FOR-Orga abgesucht, dort scheint sie (zumindest für diese Klasse) auch nicht verwendet zu werden.

Wenn gewünscht, könnte ich es aber auch in einer BC-Variante umsetzen, der Cache-Key würde sich dann halt in der Form unterscheiden je nachdem ob DB=1 oder DB!=1.

Reply:
Ich habe nun doch einen BC-Layer eingebaut

Reply:
Ok danke. Jetzt noch ein test für die neue notation, dann passt es

Reply:
Ist drin